.travis.yml

```diff
@@ -1,11 +1,32 @@
-language: node_js
-
-node_js:
-  - "6"
+language: objective-c
+osx_image: xcode7
```

Comment:
Will need to, I didn't changed this yet, I'm still looking on how to fix the test suite Appium 1.5.3 is the current latest released version, I think `v1.6.0-beta1` of Appium might supports xcode 8.